public-api/libproject/devstart.rb

```diff
@@ -583,6 +583,11 @@ def generate_public_cdr_counts(cmd_name, *args)
       ->(opts, v) { opts.bucket = v},
       "GCS bucket required."
     )
+    op.add_option(
+      "--search-vat [search-vat]",
+      ->(opts, v) { opts.search_vat = v},
+      "Flag to generate search table from VAT. Optional."
+    )
```

Comment:
You should add a default value here. This will allow you to remove line 49 from make-bq-data.sh